tools/changelog.main.kts

```diff
@@ -74,15 +78,29 @@ val argsKeyToValue = args
     .filter { it.contains("=") }
     .associate { it.substringBefore("=") to it.substringAfter("=") }
 
-val versionCommit = argsKeyless.getOrNull(0) ?: "HEAD"
+val commitsArg = argsKeyless.getOrNull(0) ?: "HEAD"
+
+var previousVersionCommitArg: String?
+var versionCommitArg: String
+if (commitsArg.contains("..")) {
```

Comment:
I wanted to write about the case with multiple ".." in strings and multiple finds the index of it, but since it's a script, it's not a blocker at all

Reply:
In a script it is fine